src/js/component/form.jsx

```diff
@@ -73,6 +95,8 @@ const renderers = [
   { tester: booleanToggleControlTester, renderer: BooleanToggleControl},
   { tester: booleanCheckboxControlTester, renderer: BooleanCheckboxControl},
   { tester: gutenbergObjectControlTester, renderer: GutenbergObjectRenderer},
+  { tester: gutenbergArrayControlTester, renderer: GutenbergArrayRenderer},
```

Comment:
`portedArrayRenderer` is default vanilla renderer, which is ported for debugging